layouts/_default/single.html

```diff
@@ -11,7 +11,10 @@
             {{ .Content }}
         </div>
 
-        <!-- `.Params.blog_like` is a stupid workaround but I don't see how we could inject scripts into blog posts without a custom `type` for each one. -->
+        <!-- `.Params.*_like` is a hacky workaround but I don't see how we could inject scripts into blog posts without a custom `type` for each one. -->
+        {{ if .Params.sva_like }}
+            <script src="{{ " js/sva-finder.bundle.gen.js" | absURL }}"></script>
+        {{ end }}
```

Comment:
I get that you're trying to group similar code but this really belongs in `scripts.html`. For one, imo it fits better there and, more importantly, the script will otherwise block the rest of the page load.